clustering/web/undertow/src/main/java/org/wildfly/clustering/web/undertow/session/DistributableSession.java

```diff
@@ -56,15 +54,17 @@
     // These mechanisms can auto-reauthenticate and thus use local context (instead of replicating)
     private static final Set<String> AUTO_REAUTHENTICATING_MECHANISMS = new HashSet<>(Arrays.asList(HttpServletRequest.BASIC_AUTH, HttpServletRequest.DIGEST_AUTH, HttpServletRequest.CLIENT_CERT_AUTH));
     static final String WEB_SOCKET_CHANNELS_ATTRIBUTE = "io.undertow.websocket.current-connections";
+    static final String IDENTITY_CONTAINER_ATTRIBUTE = "org.wildfly.elytron.web.undertow.server.servlet.ServletSecurityContextImpl$IdentityContainer";
```

Comment:
As the class name referenced here is private API we would not automatically check on class name references like this if we perform any refactoring, I would recommend a comment in the Undertow class to point us back here.

Reply:
I've opened https://issues.redhat.com/browse/ELYWEB-124 to track this.